scylladb/java-driver
# docs: update to theme 1.8.3 and bring improvements from 3.x to set 4.x as default branch

⚠️ **Important:** Just before merging this pull request, please set `scylla-4.x` as the default branch for this repository. For more details, see [Changing the default branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch#changing-the-default-branch)

## Motivation

Updates the theme to 1.8.3 and brings improvements from 3.x to make 4.x the default branch.

## How to test

1. Build the docs locally.
2. Make sure the theme version is 1.8.3:

   ![image](https://github.com/user-attachments/assets/85c2e062-055e-4eec-b68c-a09594fe4ce3)
